### PR TITLE
[DO NOT MERGE] Compatibility tables templating

### PR DESCRIPTION
**Pull request by MatonAnthony**
Currently the compatibilty tables are generated using string interpolation and are quite complicated to understand.

The goal of this branch is to write an html template to use with a templating engine (handlebars) to separate the view from the `business` code.

_This is a highly experimental PR in an experimental repository, this a non working yet work in progress_

### CODE REVIEW

**Comment by MatonAnthony** [comment]
The next steps are:

- Separate this into
  - A base template
  - A header template
  - A body template
  - A legend template
- Find a proper structure for the context used to generate the HTML.

**Comment by MatonAnthony** [comment]
The next step is to finish those templates (body.html is not finished) and to write the code to generate the required contexts.

**Comment by MatonAnthony** [comment]
I have written the context for the global table and for the header.
The script now renders a header.

The next step is still to finish the body and generate its context.

File: src/renderers/templates/legend.hbs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                        <span>{{support.title}}</span>
+                        &nbsp;
+                    </abbr>
+                </span>
+            </dt>
+            <dd>{{support.title}}</dd>
+            {{else}}
+            <dt>
+                <!--call writeIcon(strings, item, '', true) -->
+                <<abbr title="{{icon.name}}" class="only-icon">
```
Comment (flaki) [inline src/renderers/templates/legend.hbs]: Note there's a typo here.

**Comment by ExE-Boss** [comment]
I implemented templating using Mozilla Nunjucks in #8, which is now complete, doesn’t have a failing build and works.

**Pull request closed without merging**